### PR TITLE
CTCP-4052

### PR DESCRIPTION
**Pull request by Reece-Carruthers**

### CODE REVIEW

**Comment by Reece-Carruthers** [comment]
Once CTCP-3960 is done and nav updated this can be reviewed

File: app/controllers/transport/border/active/AddAnotherBorderTransportController.scala [hunk 1]
```diff
@@ -52,23 +57,32 @@ class AddAnotherBorderTransportController @Inject() (
       viewModel.count match {
         case 0 =>
           Redirect(
-            controllers.transport.border.routes.BorderModeOfTransportController.onPageLoad(departureId, mode)
-          ) //TODO: Change to /add-identification when built
+            routes.IdentificationController.onPageLoad(departureId, mode, Index(0))
```
Comment (jackheslop96) [inline app/controllers/transport/border/active/AddAnotherBorderTransportController.scala:60]: The TODO seems to suggest this should be going to the /add-identification page, but this looks like the /identification page? Was the TODO incorrect? Is there a page missing?

Comment (kenneth-rayner) [reply app/controllers/transport/border/active/AddAnotherBorderTransportController.scala]: @jackheslop96 spoke to Sayak this  page can load up regardless of how many there are and  show zero if the last one was removed. Code updated, Messaged Gavin and we may have a separate ticket for dynamic content if there are zero

Comment (kenneth-rayner) [reply app/controllers/transport/border/active/AddAnotherBorderTransportController.scala]: Following discussions. Removed the Remove link for when there is only one BorderMeansOfTransport if user has more than one and removes one then confirm remove goes back to the add another page.

**Pull request closed without merging**